src/applications/disability-benefits/all-claims/tests/pages/prisonerOfWar.unit.spec.jsx

```diff
@@ -124,9 +124,10 @@ describe('Prisoner of war info', () => {
         definitions={formConfig.defaultDefinitions}
         schema={schema}
         uiSchema={uiSchema}
-        data={Object.assign({}, formData, {
+        data={{
+          ...formData,
```

Comment:
what is this?

Reply:
Ugh, I thought I changed this back.  Some strange merge conflicts when I pulled from main. Thanks